src/bin/vip-import-sql.js

```diff
@@ -38,79 +41,93 @@ const err = message => {
 	process.exit( 1 );
 };
 
+const debug = debugLib( 'vip:vip-import-sql' );
+
 command( {
 	appContext: true,
 	appQuery,
 	requiredArgs: 1, // TODO print proper usage example
 	envContext: true,
 	module: 'import-sql',
 	requireConfirm: 'Are you sure you want to import the contents of the provided SQL file?',
-} ).argv( process.argv, async ( arg, opts ) => {
-	const { app, env } = opts;
-	const [ fileName ] = arg;
-
-	const trackEventWithEnv = async ( eventName, eventProps = {} ) =>
-		trackEvent( eventName, { ...eventProps, appId: env.appId, envId: env.id } );
+} ).option( 'search-replace', 'Specify the <from> and <to> pairs to be replaced' )
+	.option( 'in-place', 'Perform the search and replace explicitly on the input file' )
+	.argv( process.argv, async ( arg, opts ) => {
+		const { app, env, searchReplace } = opts;
+		const primaryDomainName = env.primaryDomain.name;
+		const [ fileName ] = arg;
 
-	await trackEventWithEnv( 'import_sql_command_execute' );
+		const trackEventWithEnv = async ( eventName, eventProps = {} ) =>
+			trackEvent( eventName, { ...eventProps, appId: env.appId, envId: env.id } );
 
-	console.log( '** Welcome to the WPVIP Site SQL Importer! **\n' );
+		await trackEventWithEnv( 'import_sql_command_execute' );
 
-	if ( ! currentUserCanImportForApp( app ) ) {
```

Comment:
I've been meaning to pull out the `currentUserCanImportForApp` function entirely since it's pretty useless as is & should be handled by the `appQuery`, I think:

https://github.com/Automattic/vip/blob/74a834868dcf9e1dbde2d356077d6c6150ba6814/src/lib/site-import/db-file-import.js#L27-L30

...but it seems unrelated to the rest of this PR.  Can we leave this be for the purposes of this PR & address it separately, please?

Reply:
@jblz I don't think I did this intentionally.  I'm going to check and see if this resulted from that merge in from master.

Reply:
Adding this back in 👍